docs/wiki/research/PBS/mev-boost.md

```diff
@@ -0,0 +1,56 @@
+<!-- @format -->
+
+# Mev-boost: A Popular PBS Implementation
+
+[comment]: <> (This is still unrefined)
```

Comment:
Remove this comment. If changes are required, make changes and submit a PR to update.